packages/firestore/src/api/timestamp.ts

```diff
@@ -83,6 +83,14 @@ export class Timestamp {
     );
   }
 
+  _toISOString(): string {
```

Comment:
Hrm. This is no longer an ISO String, so I think we should drop that naming. Furthermore, I am not a huge fan of adding this to our public API class (even when it is meant to be private). I would suggest either inlining it into the callsite or adding another dedicated method to RemoteSerializer.

Reply:
I'll inline this but just to clarify, the result is still an ISO string, just with more digits. The standard does not specify limits on the number of digits, and the fact that JS limits it to 3 digits is more like an issue with Date resolution.